pennylane/devices/default_qubit_autograd.py

```diff
@@ -92,6 +92,7 @@ class DefaultQubitAutograd(DefaultQubit):
     _transpose = staticmethod(np.transpose)
     _tensordot = staticmethod(np.tensordot)
     _conj = staticmethod(np.conj)
+    _real = staticmethod(np.real)
```

Comment:
Minor, but we are in the process of slowly migrating the devices to `qml.math` :) Not an issue here though!